packages/@aws-cdk/aws-rds/test/serverless/integ.serverless-cluster.ts

```diff
@@ -0,0 +1,35 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as cdk from '@aws-cdk/core';
+import { DatabaseClusterEngine, ParameterGroup, ServerlessDatabaseCluster } from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-rds-integ');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 2 });
+
+const params = new ParameterGroup(stack, 'Params', {
```

Comment:
Any reason where's not relying on the default ParameterGroup here? I would think that would be a nice test to have.

Reply:
I agree, I'll drop the params